src/CarwowTheme/Filters.elm

```diff
@@ -64,16 +65,50 @@ select id label help_message options value msg =
 
 {-| Placeholder
 -}
-filterGroupItem : FilterGroupItem msg -> String -> Html.Html msg
-filterGroupItem item groupLabel =
-    li [ class "filter__input" ]
-        (filterCheckbox
-            item.filterId
-            [ Html.span [ Html.Attributes.class "filter__input-label" ] [ Html.text item.filterLabel ] ]
-            item.filterValue
-            item.message
-            (filterColoured item.filterLabel (String.toLower groupLabel))
-        )
+filterGroupItem : FilterGroupItem msg -> String -> String -> Html.Html msg
+filterGroupItem item groupLabel filterPrefix =
+    let
+        checkboxes = (filterCheckboxFromItem item groupLabel)
+        description = div [ class "filter__description" ] [text item.filterDescription]
+        content = List.concat [ checkboxes, [description] ]
+    in
+        li [ class "filter__input" ]
+            content
+
+filterGroupItemWithExpander : FilterGroupItem msg -> String -> String -> Html.Html msg
+filterGroupItemWithExpander item groupLabel filterPrefix =
```

Comment:
I don't think this function should be exposed - the existing `filterGroupItem` should choose whether or not to render the expander depending on whether `item.filterDescription` is `Nothing` or `Just description`.